packages/@lwc/ssr-compiler/src/compile-js/index.ts

```diff
@@ -205,6 +213,32 @@ const visitors: Visitors = {
     },
 };
 
+function validateDecorators(decorators: EsDecorator[]) {
+    if (decorators.length < 2) {
+        return;
+    }
+
+    const expressions = decorators.map(({ expression }) => expression);
+
+    const hasWire = expressions.some(
+        (expr) => is.callExpression(expr) && is.identifier(expr.callee, { name: 'wire' })
+    );
+
+    const hasApi = expressions.some((expr) => is.identifier(expr, { name: 'api' }));
+
+    if (hasWire && hasApi) {
+        throw generateError(DecoratorErrors.CONFLICT_WITH_ANOTHER_DECORATOR, 'api');
+    }
+
+    const hasTrack = decorators.some(({ expression }) =>
+        is.identifier(expression, { name: 'track' })
+    );
```

Comment:
```suggestion
    const hasTrack = expressions.some((expr) => is.identifier(expr, { name: 'track' });
```
Seems a bit silly to establish a pattern and then ignore it.